cms/templates/partials/program-description.html

```diff
@@ -0,0 +1,33 @@
+{% load image_version_url %}
+<section class="program-block">
+  <div class="container">
+    <div class="gray-box d-none d-sm-block">
+      <h2>{{ page.statement | safe }} </h2>
+    </div>
+    <div class="program-intro">
+      <h2><div class="description">{{ page.heading }}</div></h2>
+      <div class="info">
+        <p>{{ page.body | safe }}</p>
+      </div>
+    </div>
+    <div class="row program-content">
+      <div class="col-sm-6">
+        <div class="img-holder">
+          <img src="{% image_version_url page.image 'max-1000x500' %}" alt="Program Description Image" />
```

Comment:
rendition should be `fill-1000x500` at a minimum although IIRC we should be doing `1920` width rather than `1000` width.

Reply:
@ahmed-belal `fill-1000x500` will crop the image that will keep the UI away from the actual design.

Reply:
As per design this should be `fill-1000x1000` or `fill-960x960` (1:1 aspect ratio), size should be 540px x 540px on md-up, full-width plus 15px padding on either sides on mobile. The reason we need `fill` instead of `max` is that `max` does not respect any given aspect ratio. On landscape or portrait images the image element can change aspect ratio based on the original aspect ratio of the file itself and change the design. Example below, notice it is not rendered as a square image.

<img width="1202" alt="Screenshot 2020-06-02 at 5 32 33 PM" src="https://user-images.githubusercontent.com/6387678/83520393-0f5f4280-a4f7-11ea-8971-187b5f539bc4.png">